plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -58,6 +59,7 @@ public TestingPostgreSqlServer()
                 .withUsername(USER)
                 .withPassword(PASSWORD)
                 .withCommand("postgres", "-c", "log_destination=stderr", "-c", "log_statement=all");
+        configureFixedExposedPortForLocalEnv(dockerContainer, POSTGRESQL_PORT);
```

Comment:
`dockerContainer` already knows port numbers (`getExposedPorts`). Why do we want to set `POSTGRESQL_PORT` explicitly?

Reply:
👍🏼 I'd expose all ports if possible (without passing them explicitly)